jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/RadioButtonSection.java

```diff
@@ -0,0 +1,42 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+
+public class RadioButtonSection {
+    @Css("#basic-radio-group")
+    public static RadioButtons basicRadioButtons;
+
+    @UI("#radio-option-one")
+    public static Button firstBasicRadioButton;
+
+    @UI("#radio-option-two")
+    public static Button secondBasicRadioButton;
+
+    @Css("#season-radio-group")
+    public static RadioButtons seasonRadioButtons;
+
+    @UI("#favorite-season-winter")
+    public static Button winterNGModelRadioButton;
+
+    @UI("#favorite-season-spring")
+    public static Button springNGModelRadioButton;
+
+    @UI("#favorite-season-summer")
+    public static Button summerNGModelRadioButton;
+
+    @UI("#favorite-season-autumn")
+    public static Button autumnNGModelRadioButton;
+
+    @UI("/html/body/div[1]/div[2]/div/radio-ng-model-example/div")
+    public static Text favoriteSeason;
+
+    public static String enableSeasonButton = "mat-radio-button example-radio-button mat-accent cdk-focused cdk-mouse-focused mat-radio-checked";
```

Comment:
The class name is too long. Maybe it's better to use a significant part of the class name and Matcher in method attr(String attrName, Matcher<String> condition)